src/publisher/ajson_ingestor.py

```diff
@@ -78,7 +78,7 @@ def _ingest(data, force=False):
         # only update the fragment if this article version has *not* been published *or* if force=True
         update_fragment = not av.published() or force
         fragments.add(av, XML2JSON, data['article'], pos=0, update=update_fragment)
-        fragments.set_article_json(av, quiet=True)
+        fragments.set_article_json(av, quiet=force) # article-json validation occurs here
```

Comment:
so if `force` is specified the article would still be ingested? Silent corrections do that, but we want to validate the JSON on ingest for them too, to catch the problem as soon as possible